src/app/vault/ciphers.component.ts

```diff
@@ -80,6 +86,90 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
         this.didScroll = this.pagedCiphers.length > this.pageSize;
     }
 
+    async loadOranizations(): Promise<Map<string, string>> {
+        const organizations = await this.userService.getAllOrganizations();
+        const organizationNames = new Map<string, string>();
+
+        if (organizations.length > 0) {
+            organizations.forEach(organization => {
+                organizationNames.set(organization.id, organization.name);
+            });
+        }
+
+        return organizationNames;
+    }
+
+    getOrganizationById(c: CipherView): string {
+        if (c.organizationId != null) {
+            return this.organizationNames.get(c.organizationId);
+        }
+        return 'Personal';
+    }
+
+    sortCiphers(sortBy: string): void {
+        if (this.sortByOptions.includes(sortBy)) {
+            if (sortBy === this.sortBy) {
+                this.sortedDescending = !this.sortedDescending;
+            } else {
+                this.sortBy = sortBy;
+                this.sortedDescending = true;
+            }
+
+            const self = this;
+
+            if (sortBy === 'lastEdited') {
+                this.ciphers.sort((a: CipherView, b: CipherView) => {
+                    if (self.sortedDescending) return new Date(b.revisionDate).valueOf() - new Date(a.revisionDate).valueOf(); // weird typescript thing: https://stackoverflow.com/a/60688789/7672369
+                    else return new Date(a.revisionDate).valueOf() - new Date(b.revisionDate).valueOf();
```

Comment:
Please don't use single line if/return statement. Add curly braces
```suggestion
                    if (self.sortedDescending) {
                       return new Date(b.revisionDate).valueOf() - new Date(a.revisionDate).valueOf(); // weird typescript thing: https://stackoverflow.com/a/60688789/7672369
                    }
                    else {
                        return new Date(a.revisionDate).valueOf() - new Date(b.revisionDate).valueOf();
                    }
```